lib/cisco_node_utils/cmd_ref/span_session.yaml

```diff
@@ -0,0 +1,64 @@
+# SPAN session
```

Comment:
Switch Port Analyzer (SPAN) is the feature. `monitor session` is just the CLI stanza so I didn't want to conflate the two. Just seems cleaner and prevents confusion if `monitor session` becomes a different feature altogether in NXOS. Open to feedback on that though and can be easily changed back.